openfisca_core/projectors/typing.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Iterable, Protocol
+
+
+class Entity(Protocol):
+    @property
+    def is_person(self) -> bool:
+        ...
+
+
+class GroupEntity(Protocol):
+    @property
+    def containing_entities(self) -> Iterable[str]:
```

Comment:
@benjello This corresponds to the `key` of other entities?

Reply:
yes